### ISSUE TITLE
Welcome page always shows on startup

### ISSUE THREAD

**Issue by Nicras** [shanalikhan/code-settings-sync#1074]
🐛 As title says, every time I launch VSCode, the Welcome page shows up, regardless of the checkmark at the bottom right "Show this page on startup".

Thanks in advance

🌴 **Visual Studio Code Version :** 1.39.2
🌴 **Code Settings Sync Version :** 3.4.3
🌴 **Standard or Insiders :** Standard
🌴 **Portable or Installed :** Installed
🌴 **OSS or Official Build :** Official
🌴 **Operating System :** Windows 10
🌴 **Occurs On:** Startup
🌴 **Proxy Enabled:** No
🌴 **Gist Id:** c22cafa8eddd6b219b20ef294c41b78d

📰 **To Reproduce**

💪 **Expected behavior**

📺 **Console Error Log** 
If applicable, add screenshots or paste the console log from developer console of VSCode (Help Menu > Developer Tool - Console Tab). 

📺  **Additional context**

**Comment by ManPython** [comment]
Right.. it's annoying. Should exist some option to turn off this or in #942 VSC globaly. Lets say that any plugin hvae this and before work firstly we must close all news about updates/features.
It's beter to do some "sumary list of extensions changes" in VSC. Can be useful in case that after restart of VSC this cheking updates and updating many ext.
https://github.com/microsoft/vscode/issues/82537

**Comment by shanalikhan** [comment]
> As title says, every time I launch VSCode, the Welcome page shows up, regardless of the checkmark at the bottom right "Show this page on startup".

Have you tried turning off the checkbox ? Does it still opens at startup.

>  If applicable, add screenshots or paste the console log from developer console of VSCode (Help Menu > Developer Tool - Console Tab).

Can you post any developer console log, if there is any

**Comment by DonaldMerizaj** [comment]
Same problem with v3.4.3 on Ubuntu, even after unchecking the checkbox it keeps showing on startup ! Any update on this ?

**Comment by thedoglett** [comment]
Same here. 

I have tried checking and unchecking the 'Show this page on startup' checkbox but every time I open Visual Studio Code the window keeps showing.

This has happened on an iMac 27" and MacBook Pro 16" 2020, both running Catalina.

**Comment by snazemi** [comment]
Same here. 
Windows 2019 (Windows_NT x64 10.0.17763) 
VS Code Version: 1.51.1 (system setup)
Commit: e5a624b788d92b8d34d1392e4c4d9789406efe8f
Electron: 9.3.3
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
Settings Sync (shan.code-settings-sync): v3.4.3

**Comment by thedoglett** [comment]
> Same here.
> Windows 2019 (Windows_NT x64 10.0.17763)
> VS Code Version: 1.51.1 (system setup)
> Commit: e5a624b788d92b8d34d1392e4c4d9789406efe8f
> Electron: 9.3.3
> Chrome: 83.0.4103.122
> Node.js: 12.14.1
> V8: 8.3.110.13-electron.0
> Settings Sync (shan.code-settings-sync): v3.4.3

I've turned it off now, just too annoying.

**Comment by diyxiaoshitou** [comment]
为什么一直关闭不了啊，太烦人了

**Comment by anthonyhtang** [comment]
Same here.

**Comment by Stradivario** [comment]
Cannot make it to disappear even with unchecked "show this page on startup"

**Comment by phillip-haydon** [comment]
1 year 4 months...

**Comment by christianhengst** [comment]
Same here on my MacBook Air M1 and (older) iMac. A fix would be awesome! Thanks!

**Comment by danilo-delbusso** [comment]
Same on Ubuntu 20.04.2 LTS 64-bit.

How's this still an issue? It's been almost 1.5 years

**Comment by ItsMeKevin1227** [comment]
Same here, MacBook pro M1.

**Comment by amirsymbiote** [comment]
@shanalikhan It has been almost 2 years with this issue. Can you please take a look at resolving this? On start up whether the 'Show this page on startup' checkbox is ticked or not it continues to show on start up. I'm having this issue on Ubuntu 21.04, looks like this issue is not platform specific though.

**Comment by Etto91** [comment]
same here on macbook pro 13 intel

**Comment by omarbangla24** [comment]
Still same

**Comment by diyxiaoshitou** [comment]
It's been two years. Why not fix it

**Comment by QueenKunkun** [comment]
still here

**Comment by ghost** [comment]
It's still here

**Versions**
OS: macOS 12.0.1
Visual Studio Code: 1.62.3
Extension: 3.4.3

**Comment by jfletcher-nuxeo** [comment]
The package hasn't been updated since 2019. Seems unlikely this will be fixed unless one of us does it :)

**Comment by Raul-Gallardo** [comment]
I had to uninstall it and install it again, it worked.

**Comment by andresprimera** [comment]
Don't know if it would help but i just find out this library is deprecated and we should us the build in functionality. 

https://code.visualstudio.com/docs/editor/settings-sync

Hope it helps.
